.gitignore

```diff
@@ -33,7 +33,7 @@ public/system/project_attachments/
 .yardoc/
 .rubocop-https---raw-githubusercontent-com-PublicHealthEngland-ndr-dev-support-master--rubocop-yml
 public/assets/*
-
+vendor/cache/*
```

Comment:
@FrancescoNea - why is `vendor/cache` being git ignored?

Reply:
Hi @ollietulloch , I have removed `vendor/cache` from the list of ignored folders in `.gitignore`